inspirehep/cli/oauth.py

```diff
@@ -0,0 +1,52 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# inspirehep is free software; you can redistribute it and/or modify it under
+# the terms of the MIT License; see LICENSE file for more details.
+
+
+import click
+from flask import current_app
+from flask_security.utils import hash_password
+from invenio_accounts.models import Role
+from invenio_db import db
+from invenio_oauth2server.models import Client, Token
+
+
+def init_oauth_token():
+    ds = current_app.extensions["invenio-accounts"].datastore
+    admin = Role.query.filter_by(name="admin").one()
+    with db.session.begin_nested():
+        user = ds.create_user(
+            email="admin@inspirehep.net",
+            password=hash_password("123456"),
+            active=True,
+            roles=[admin],
+        )
+    db.session.commit()
+    click.secho("User admin added successfully", fg="green")
+    with db.session.begin_nested():
+        c = Client(
```

Comment:
it's better to use clear names like ``client = `` ``token =`` it's easier to read